src/main/java/com/dreamhorizon/enchantments/enchantments/LifeLeech.java

```diff
@@ -30,27 +30,31 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 
+/**
+ * @author Collin Herber
+ * @version 1.0
+ */
 public class LifeLeech {
     public static DHEnchantment getEnchantmentInformation() {
         return new DHEnchantment(
-            new NamespacedKey(DHCore.getPlugin(DHCore.class), "lifeleech"),
-            "Life Leech",
-            1,
-            2,
-            EnchantmentTarget.WEAPON,
-            false,
-            false,
-            new ArrayList<>(),
-            Arrays.asList(
-                Material.WOODEN_SWORD,
-                Material.STONE_SWORD,
-                Material.IRON_SWORD,
-                Material.GOLDEN_SWORD,
-                Material.DIAMOND_SWORD
-            )
+                new NamespacedKey(DHCore.getPlugin(DHCore.class), "lifeleech"),
+                "Life Leech",
+                1,
+                2,
+                EnchantmentTarget.WEAPON,
+                false,
+                false,
+                new ArrayList<>(),
+                Arrays.asList(
```

Comment:
We need to sync on our intellij formatting options, not going to have this battle each time.

Reply:
Sure, if I can use the auto-format to have it in your syntax it's fine.